README.md

```diff
@@ -78,20 +62,44 @@ SSL_CERT: /tmp/ssl-cert.pem
 ```
 
 #### Push with `meta-buildpack`
+**For custom buildpacks see [below](#custom-buildpack-instructions)**
 
-As described in the [`meta-buildpack` documentation](https://github.com/cf-platform-eng/meta-buildpack#how-it-works) the Conjur decorate buildpack will **ONLY** be invoked in the following scenarios:
+As described in the [`meta-buildpack` documentation](https://github.com/cf-platform-eng/meta-buildpack#how-it-works) the Conjur decorate buildpack will **ONLY** be invoked in the following are true:
 
 + `cf push` with `meta-buildpack` installed and at the top of list of buildpacks. 
-+ `cf push` with the `meta-buildpack` specified in app's manifest.
-+ `cf push -b meta-buildpack`
++ no buildpack is specified in app's manifest.
++ no `-b` flag value is provided to the CLI. 
 
 Assuming the app is bound to a Conjur service instance, pushing the app will result in the Conjur buildpack being invoked, as well as the language buildpack. The secrets specified in the `secrets.yml` file will now be available in the session environment variables at the start of the app.
 
-### Alternative usage without `meta-buildpack`
+### <a name="custom-buildpack-usage"></a>Usage for users with custom buildpacks
 
-**This section requires that the app be bound to a Conjur service instance and the `secrets.yml` file has been created and is in place.**
+To retrieve secrets without using `meta-buildpack` you can simply:
+
++ copy the contents of the `./lib` directory in this repository to the `./.profile.d` directory relative to your app's root directory.
  
-To circumvent the limitations of `meta-buildpack` you can simply copy the contents of the `./lib` directory in this repository to the `./.profile.d` directory relative to your app's root directory. When your application starts the secrets specified in the `secrets.yml` file will now be available in the session environment variables at the start of the app.
+When your application starts the secrets specified in the `secrets.yml` file will now be available in the session environment variables at the start of the app.
+
+## How does the buildpack work ?
```

Comment:
Can we move this whole section up so that it's below the `Requirements` section? It has a lot of good info and it would be great if it were higher on the page so that people could see it